webextensions/javascript-apis.json

```diff
@@ -3949,7 +3949,7 @@
                     "version_added": "53.0"
                   },
                   "opera": {
-                    "version_added": false
+                    "version_added": true
```

Comment:
The Opera docs say "Opera only supports the launchWebAuthFlow() Method in this API." (https://dev.opera.com/extensions/apis/). Are they wrong here?